pages/[lng]/register/index.tsx

```diff
@@ -55,10 +55,21 @@ const RegisterPage: FC<any> = ({
   hasFacebookAuth
 }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
   const i18n: any = useI18n()
+  const recaptchaRef = useRef<any>()
 
   const [isVerified, setIsVerified] = useState<boolean>(false)
   const linksBreadcrumb = [`${i18n.t("header.home")}`, `${i18n.t("register.title")}`]
```

Comment:
```suggestion
  const linksBreadcrumb = [i18n.t("header.home"), i18n.t("register.title")]
```

dibeginikan jika memungkinkan